Sources/TuistKit/Commands/GenerateCommand.swift

```diff
@@ -41,7 +41,7 @@ public struct GenerateCommand: AsyncParsableCommand, HasTrackableParameters {
         help: "Don't open the project after generating it.",
         envKey: .generateOpen
     )
-    var open: Bool = true
+    var open: Bool = !CIChecker().isCI()
```

Comment:
Generally, this is a bit of an antipattern but in this case, I think it's fine

Reply:
Yeah I thought if it is necessary to add a var for `CIChecking` here and use that so that the tests could mock it, but then noticed that we dont initialise this command with parameters so went with this approach.